### PR TITLE
Added consistency in font sizing

### PR DESCRIPTION
**Pull request by Mubashar1009**
## Change Request

Font Sizes of the text  are different in different sections  in the project, add consistency in the font sizing so UI should be looking  good 

### Implementation

* Changed font size of the inputs according to the side bar items's text font size
* Changed font size of the date picker, labels , form labels according to the sidebar  items's text  font size
* Changed font size of  Tables List heading according to the sidebar  section heading and breadcrumbs section text

### Resources
image
image

### CODE REVIEW

**Review by Monfernape — APPROVED** [review]

**Review by muhammad-bilal-ali-khokhar — APPROVED** [review]

File: src/app/members/components/MemberList.tsx [hunk 1]
```diff
@@ -133,7 +133,7 @@ export const MemberList = ({ members }: { members: Props }) => {
               <TableRow>
                 <TableCell
                   colSpan={columns.length}
-                  className="bg-gray-300 text-gray-700 text-left px-4 py-2 font-bold"
+                  className="bg-gray-300 text-gray-700 text-left px-4 py-2 font-semibold text-sm"
```
Comment (nadirhussain786) [inline src/app/members/components/MemberList.tsx:136]: @Mubashar1009  font size should be `xs`

File: src/common/FormTitle/FormTitle.tsx [hunk 2]
```diff
@@ -2,8 +2,6 @@ import React from "react";
 
 export const FormTitle = ({ title }: { title: string }) => {
   return (
-    <h1 className="text-sm font-semibold text-gray-800 my-4">
-      {title}
-    </h1>
+    <h1 className="text-base font-semibold text-gray-800 my-4">{title}</h1>
```
Comment (nadirhussain786) [inline src/common/FormTitle/FormTitle.tsx:5]: @Mubashar1009 for font size should be `sm`.

File: src/components/ui/label.tsx [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+import * as LabelPrimitive from "@radix-ui/react-label";
+import { cva, type VariantProps } from "class-variance-authority";
 
-import { cn } from "@/lib/utils"
+import { cn } from "@/lib/utils";
 
 const labelVariants = cva(
-  "text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
-)
+  "text-sm font-semibold leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
```
Comment (nadirhussain786) [inline src/components/ui/label.tsx:10]: @Mubashar1009 font should be `font-medium`.

File: src/components/ui/form.tsx [hunk 4]
```diff
@@ -95,7 +95,7 @@ const FormLabel = React.forwardRef<
   return (
     <Label
       ref={ref}
-      className={cn("text-xs font-semibold", className)}
+      className={cn("text-sm font-semibold", className)}
```
Comment (nadirhussain786) [inline src/components/ui/form.tsx:98]: @Mubashar1009 font size should be `xs` for label

File: src/components/ui/button.tsx [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
+import * as React from "react";
+import { Slot } from "@radix-ui/react-slot";
+import { cva, type VariantProps } from "class-variance-authority";
 
-import { cn } from "@/lib/utils"
+import { cn } from "@/lib/utils";
 
 const buttonVariants = cva(
-  "inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0",
+  "inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-semibold transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:size-4 [&_svg]:shrink-0",
```
Comment (nadirhussain786) [inline src/components/ui/button.tsx:8]: @Mubashar1009 font weight should be remain `font-medium`

**Review by nadirhussain786 — CHANGES_REQUESTED** [review]

**Review by nadirhussain786 — APPROVED** [review]

**Pull request merged**